logos/logos-workernode/logos_worker_node/logos_bridge.py

```
@@ -627,53 +914,71 @@ def _run_calibration() -> None:
+            terminal_event = "calibration_session_cancelled"
+        finally:
+            session.current_model = None
+            self._record_calibration_event(
+                terminal_event,
+                details=f"sleep_level={session.sleep_level}",
+            )
+            if lane_manager is not None:
+                try:
+                    lane_manager._mark_status_dirty()  # noqa: SLF001
+                except Exception:  # noqa: BLE001
+                    pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)